Source/Krypton Components/Krypton.Toolkit/General/OutlookGrid/Formatting/ColorFormatting.cs

```diff
@@ -60,7 +60,7 @@ public static double ConvertBar(double value, double min, double max)
         /// <returns></returns>
         public static Color ConvertTwoRange(double value, double min, double max, TwoColorsParams? par)
         {
-            ColorHelper.HSVColor a = ColorToHSV(par.MinimumColor);
+            ColorHelper.HSVColor a = ColorToHSV(par!.MinimumColor);
```

Comment:
Make par non nullable, as you have only checked it's first usage.

Reply:
Done